dev/Pager/PrototypePager/Themes/PrototypePager.xaml

```diff
@@ -0,0 +1,181 @@
+                                        <Setter Target="NextPageButton.IsEnabled" Value="False"/>
+                                    </VisualState.Setters>
+                                </VisualState>
+                            </VisualStateGroup>
+                            <VisualStateGroup x:Name="LastPageButtonStates">
+                                <VisualState x:Name="LastPageButtonVisible"/>
+                                <VisualState x:Name="LastPageButtonCollapsed">
+                                    <VisualState.Setters>
+                                        <Setter Target="LastPageButton.Visibility" Value="Collapsed"/>
+                                        <Setter Target="LastPageButton.IsEnabled" Value="False"/>
+                                    </VisualState.Setters>
+                                </VisualState>
+                            </VisualStateGroup>
+                            <VisualStateGroup x:Name="PagerDisplayModeStates">
+                                <VisualState x:Name="NumberBoxVisible">
+                                    <VisualState.Setters>
+                                        <Setter Target="NumberBoxDisplay.IsEnabled" Value="True"/>
+                                        <Setter Target="BoxPanels.Visibility" Value="Visible"/>
+                                        <Setter Target="NumberBoxDisplay.Visibility" Value="Visible"/>
+                                    </VisualState.Setters>
+                                </VisualState>
+                                <VisualState x:Name="ComboBoxVisible">
+                                    <VisualState.Setters>
+                                        <Setter Target="ComboBoxDisplay.IsEnabled" Value="True"/>
+                                        <Setter Target="BoxPanels.Visibility" Value="Visible"/>
+                                        <Setter Target="ComboBoxDisplay.Visibility" Value="Visible"/>
+                                    </VisualState.Setters>
+                                </VisualState>
+                                <VisualState x:Name="NumberPanelVisible">
+                                    <VisualState.Setters>
+                                        <Setter Target="BoxPanels.Visibility" Value="Collapsed"/>
+                                        <Setter Target="NumberBoxDisplay.Visibility" Value="Collapsed"/>
+                                        <Setter Target="ComboBoxDisplay.Visibility" Value="Collapsed"/>
+                                        <Setter Target="NumberPanelDisplay.Visibility" Value="Visible"/>
+                                    </VisualState.Setters>
+                                </VisualState>
+                            </VisualStateGroup>
+                        </VisualStateManager.VisualStateGroups>
+                        <!--Spacing="{TemplateBinding Spacing}"-->
```

Comment:
>Spacing [](start = 28, length = 7)

Should this get added back in or removed?

Reply:
I removed it for now